### PR TITLE
Energy related fixes

### PR DESCRIPTION
**Pull request by Yung-Beef**

### CODE REVIEW

File: docs/basics/faq/Overview.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ You can use your SUB tokens to create a space, make posts, and interact with oth
 by following them, or commenting and voting on their posts. Currently, you can also tip other users using your SUB tokens, 
 and more monetization features will be added in the future.
 
-You can also burn SUB to generate a larger amount of energy to complete transations.
+You can also burn SUB to generate a energy, allowing you to complete transations with reduced fees.
```
Comment (F3Joule) [inline docs/basics/faq/Overview.md:73]:
```suggestion
You can also burn SUB to generate an energy, allowing you to complete transations with reduced fees.
```

File: docs/basics/lightpaper/architecture/Energy.md [hunk 2]
```diff
@@ -4,16 +4,16 @@ title: Energy
...
+Energy allows usage of the Subsocial network without tokens, and has a few key points and implications. 
+Energy is not actually a token, and cannot be transferred between accounts, and transaction fees are lower when using energy.
 To create energy, tokens need to be burned, but the energy can be created in a different account than the one burning the tokens.
 
-For example, Alive burns 10 SUB and creates 11 NRG in her account. 
-This allows Alice to perform more actions than if she had used her SUB tokens for the fees, so she is incentivized to convert her tokens into NRG, 
-creating deflationary pressure on the token.
+The energy coefficient is an on-chain parameter that can be set by the chain's governance. This coefficient determines how much cheaper transactions are when energy is used instead of SUB. At the time of writing, the energy coefficient is 5x, meaning transaction fees are 5x cheaper if energy is used.
+
+For example, Alice burns 10 SUB and creates 10 NRG in her account.
```
[the remark this answers is not in the document]
Comment (F3Joule) [reply docs/basics/lightpaper/architecture/Energy.md:13]: Maybe also "for her account"?

File: docs/basics/lightpaper/architecture/Energy.md [hunk 3]
```diff
@@ -4,16 +4,16 @@ title: Energy
...
-This allows Alice to perform more actions than if she had used her SUB tokens for the fees, so she is incentivized to convert her tokens into NRG, 
-creating deflationary pressure on the token.
+The energy coefficient is an on-chain parameter that can be set by the chain's governance. This coefficient determines how much cheaper transactions are when energy is used instead of SUB. At the time of writing, the energy coefficient is 5x, meaning transaction fees are 5x cheaper if energy is used.
+
+For example, Alice burns 10 SUB and creates 10 NRG in her account. 
+This allows Alice to perform roughly 2500 actions by using energy, instead of only 500 actions by using SUB tokens. 
+Thus, Alice is incentivized to convert her tokens into NRG, creating deflationary pressure on the token.
 
-Alice could instead burn those 10 tokens and create the 11 NRG in Bob's account, allowing Bob to use Subsocial without ever getting SUB tokens.
+Alice could instead burn those 10 tokens and create the 10 NRG in Bob's account, allowing Bob to use Subsocial without ever getting SUB tokens.
```
Comment (F3Joule) [inline docs/basics/lightpaper/architecture/Energy.md:17]:
```suggestion
Alice could instead burn those 10 tokens and create the 10 energy in Bob's account, allowing Bob to use Subsocial without ever getting SUB tokens.
```

Comment (F3Joule) [reply docs/basics/lightpaper/architecture/Energy.md:17]: The same "for" suggestion here.

**Review by F3Joule — CHANGES_REQUESTED** [review]

File: docs/basics/faq/Overview.md (hunk 1 above)
Comment (Yung-Beef) [reply docs/basics/faq/Overview.md]: Removed it entirely, thanks for pointing it out
"You can also burn SUB to generate energy"

**Review by F3Joule — APPROVED** [review]

**Pull request merged**